components/calendar/data/data/mod.rs

```diff
@@ -46,8 +27,6 @@ macro_rules! __impl_any_provider {
         }
     };
 }
-#[doc(inline)]
-pub use __impl_any_provider as impl_any_provider;
 #[clippy::msrv = "1.66"]
 pub struct BakedDataProvider;
 impl_data_provider!(BakedDataProvider);
```

Comment:
Also I just noticed that we're exporting a provider from these crates that we aren't using anywhere.

Reply:
no we're not, we're including macros.rs, not mod.rs